flow/scenarios/base_scenario.py

```diff
@@ -109,7 +109,11 @@ def __init__(self,
             self.connections = None
 
         # specify routes vehicles can take
-        self.routes = self.specify_routes(net_params)
```

Comment:
i think we will eventually want to add routes via `specify_routes`. Can we just ignore this? As of #486 this should be producing default values, and should not break anything.

Reply:
indeed it is not necessary anymore, I removed it.